test/old/testdir/runnvim.vim

```diff
@@ -23,11 +23,13 @@ function Main()
   set lines=25
   set columns=80
   enew
-  let job = termopen(args, s:logger)
+  " FIXME: using termopen() hangs on Windows CI
+  let job = has('win32') ? jobstart(args, s:logger) : termopen(args, s:logger)
```

Comment:
hmmm... what if we use jobstart() for all platforms here? Do we lose something?

Reply:
I will test.

Reply:
Nope

```
Failures: 
	From test_writefile.vim:
	Found errors in Test_writefile_sync_dev_stdout():
	Caught exception in Test_writefile_sync_dev_stdout(): Vim(call):E482: Can't open file /dev/stdout for writing: no such device or address @ command line..script /home/runner/work/neovim/neovim/test/old/testdir/runtest.vim[437]..function RunTheTest[44]..Test_writefile_sync_dev_stdout, line 4
```